src/components/navigation/VolunteerNavMenu.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState, createElement } from 'react';
+import { Link } from 'react-router-dom';
+import { Menu, Layout, Button } from 'antd';
+import {
+  ProfileOutlined,
+  TableOutlined,
+  DashboardOutlined,
+  FormOutlined,
+  MenuUnfoldOutlined,
+  MenuFoldOutlined,
+} from '@ant-design/icons';
+import Logo from '../../assets/Logo.png';
+import '../NavMenu/NavMenu.module.css';
+
+const { Sider } = Layout;
+
+const VolunteerNavMenu = () => {
+  const [collapsed, setCollapsed] = useState(false);
+  const [menuTitle, setMenuTitle] = useState('Abound Food Care');
+
+  const handleToggle = () => {
+    setCollapsed(!collapsed);
+    if (!collapsed) {
+      setMenuTitle('');
+    } else {
+      setMenuTitle('Abound Food Care');
+    }
+  };
+
+  return (
+    <div style={{ position: 'relative' }}>
+      <Sider
+        trigger={null}
+        collapsible
+        collapsed={collapsed}
+        style={{ height: '100vh', backgroundColor: 'var(--eden)', color: 'white' }}
```

Comment:
`var(--eden)` (and other css vars, for that fact) doesn't work on inline styles which is why i recommend moving this to a module.css file if you can. otherwise just use `#115740` directly